docs/Repository-Onboarding.md

```diff
@@ -0,0 +1,75 @@
+# VMR Repository Onboarding Guide
+
+This document provides a step-by-step guide for onboarding repositories into the VMR (Virtual Monolithic Repository).
+For background on VMR concepts and architecture, see [VMR Design and Operation](./VMR-Design-And-Operation.md).
+
+## Prerequisites
+
+Before onboarding a repository to the VMR, ensure the following are met:
+
+- Repository is a dependency of the .NET product (e.g. a dependency of the SDK)
+- Repository is onboarded to the [Arcade SDK](https://github.com/dotnet/arcade/blob/main/Documentation/StartHere.md) and its builds are registered in [BAR](https://github.com/dotnet/arcade-services/blob/main/docs/Darc.md#locating-the-bar-build-id-for-a-build).
+  - If not, see if it can be added via an [external source-build-reference-package](https://github.com/dotnet/source-build-reference-packages/blob/main/README.md#external)
+- Repository is compatible with [source build](https://github.com/dotnet/source-build/blob/main/Documentation/sourcebuild-in-repos/README.md)
+- Open an [issue](https://github.com/dotnet/dotnet/issues/new/choose) tracking this work and mention [@dotnet/product-construction](https://github.com/orgs/dotnet/teams/product-construction)
+
+## Onboarding Steps
+
+### Step 1: Configure Source Mappings
+
+Add your repository to `/src/source-mappings.json` and open a PR to merge the change.
+For detailed configuration options, see [Repository Source Mappings](./VMR-Full-Code-Flow.md#repository-source-mappings).
+
+### Step 2: Define the Darc Code Flow Subscriptions
+
+Define the appropriate Darc [code flow subscriptions](./Codeflow-PRs.md).
+After defined manually trigger the forward flow subscription via `darc` or [Maestro](https://maestro.dot.net/subscriptions).
+This will open a [VMR PR](https://github.com/dotnet/dotnet/pulls?q=is%3Apr+is%3Aopen+%22Source+code+updates%22) that brings in the repo's source code.
+
+Before merging, ensure the binary and license policies are satisfied:
+
+#### Step 2a: Ensure no Checked in Binaries
+
+The VMR has restrictions on checked-in binaries.
+PR and CI validation exists to ensure the [binary policy](./VMR-Permissible-Sources.md#binary-policy) is satisfied.
+If the code flow PR is green, the repo complies with the binary policy.
+
+#### Step 2b: Validate OSS Compliant Licenses
+
+The VMR has restrictions on allowed licenses.
+Before the code flow PR is merged, queue a run of the [license scan pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=1301&_a=summary) (internal Microsoft link) to validate the repo complies with the [license policy](./VMR-Permissible-Sources.md#license-policy).
```

Comment:
This should be targeting the unofficial pipeline since this will need to be done on a private branch.

```suggestion
Before the code flow PR is merged, queue a run of the [license scan pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=1490) (internal Microsoft link) to validate the repo complies with the [license policy](./VMR-Permissible-Sources.md#license-policy).
```